# INSTALL_LuMP_Pulsar_PSRXML_P0.patch
# patch file 1 for PSRXML
# 2014 Oct 08  James Anderson  --- GFZ start
#
# Copyright (c) 2014 James M. Anderson <anderson@gfz-potsdam.de>
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
#
# Note that the PSRXML software in this patch is presumably covered by a different
# copyright.  See http://www.pulsarastronomy.net/wiki/Software/PsrXML
# for more information about PSRXML copyright issues.
#
################################################################################
diff -Naur psrxml/psrxml-master/src/psrxml/psrxml.h psrxml_new/psrxml-master/src/psrxml/psrxml.h
--- psrxml/psrxml-master/src/psrxml/psrxml.h	2014-10-08 09:19:28.865408497 +0200
+++ psrxml_new/psrxml-master/src/psrxml/psrxml.h	2014-10-08 09:24:07.403939869 +0200
@@ -1,6 +1,5 @@
 #ifndef PSRXML_H_
 #define PSRXML_H_
-#include "config.h"
 
 #ifndef NAN
 #define NAN (0.0/0.0)
